lib/Optimizer/CodeGen/ConvertToQIR.cpp

```diff
@@ -204,6 +204,43 @@ void cudaq::opt::initializeTypeConversions(LLVMTypeConverter &typeConverter) {
   cudaq::opt::populateCCTypeConversions(&typeConverter);
 }
 
+/// @brief Compute function argument size in bytes and offsets
+/// of all the arguments that satisfy the filter.
+/// Start computing offsets and size from `startingArgIdx`.
+std::pair<std::size_t, std::vector<std::size_t>>
+cudaq::opt::getFunctionArgumentLayout(mlir::ModuleOp module,
```

Comment:
```suggestion
cudaq::opt::getFunctionArgumentLayout(ModuleOp module,
```
I believe this file has a `using namespace mlir;` near the top. So please delete all the `mlir::` namespace prefixes for consistency with the rest of the code.